app/views/notes/_comment.html.erb

```diff
@@ -64,7 +64,7 @@
   </script>
   
   <div id="c<%= comment.cid %>show">
-    <p><%= raw sanitize(auto_link(RDiscount.new(comment.body).to_html)) %></p>
+    <p><%= raw sanitize (RDiscount.new(title_suggestion(comment)).to_html), attributes: %w(class style href data-method src) %></p>
```

Comment:
This starts to get quite long. See how we've batched these in this code?

https://github.com/ViditChitkara/plots2/blob/624aa873794f0d904fb44e0772a2fddf51309d5d/app/helpers/application_helper.rb#L30-L42

What do you think about making this more readable with an additional helper somehow, rather than a long in-line code block?

Looking good though!

Reply:
Some helper method like:- comment_body
It could look like this:- 
```
def comment body(comment)
  RDiscount.new(title_suggestion(comment)).to_html), attributes: %w(class style href data-method src
end
```
Then we could use it in line 67 as:- 
```
<p><%= raw sanitize(auto_link(RDiscount.new(comment_body(comment)).to_html)) %></p>

```
Will this be good?